{"repository":"https://github.com/apyb/associados","dockerfilePath":"Dockerfile","startTime":1672771538159,"clone":{"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/apyb/associados'...\n","error":null,"commit":"63439044eae31d442d17549ccc3121de6d0cee80"},"originalBuild":{"startTimestamp":1672771540344,"endTimestamp":1672771592530,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t associados:latest -f /tmp/dinghy-analysis/apyb/associados/Dockerfile ."},"stdout":"Sending build context to Docker daemon   2.21MB\r\r\nStep 1/6 : FROM python:3.7.12\n3.7.12: Pulling from library/python\ne4d61adff207: Already exists\n4ff1945c672b: Already exists\nff5b10aec998: Already exists\n12de8c754e45: Already exists\nada1762e7602: Already exists\n2f2b2e030155: Already exists\n8752dc84acc9: Pulling fs layer\n9bc84992c7bf: Pulling fs layer\n356f512becef: Pulling fs layer\n9bc84992c7bf: Verifying Checksum\n9bc84992c7bf: Download complete\n356f512becef: Verifying Checksum\n356f512becef: Download complete\n8752dc84acc9: Verifying Checksum\n8752dc84acc9: Download complete\n8752dc84acc9: Pull complete\n9bc84992c7bf: Pull complete\n356f512becef: Pull complete\nDigest: sha256:b48983bebd0fe1c09639fa008e4cb51aac6277af6c6762fc58ac9d2cb7fc24ef\nStatus: Downloaded newer image for python:3.7.12\n ---> b5b74728b4f4\nStep 2/6 : ENV PYTHONUNBUFFERED=1\n ---> Running in 61fcb1119e57\nRemoving intermediate container 61fcb1119e57\n ---> d271e7c1449b\nStep 3/6 : COPY requirements_test.txt requirements.txt /tmp/\n ---> 9d7d73f090fc\nStep 4/6 : RUN pip install --upgrade pip && pip install -r /tmp/requirements_test.txt\n ---> Running in 8a633591c25b\nRequirement already satisfied: pip in /usr/local/lib/python3.7/site-packages (21.2.4)\nCollecting pip\n  Downloading pip-22.3.1-py3-none-any.whl (2.1 MB)\nInstalling collected packages: pip\n  Attempting uninstall: pip\n    Found existing installation: pip 21.2.4\n    Uninstalling pip-21.2.4:\n      Successfully uninstalled pip-21.2.4\nSuccessfully installed pip-22.3.1\n\u001b[91mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n\u001b[0mCollecting django_s3_folder_storage\n  Cloning https://github.com/pythonbrasil/django-s3-folder-storage.git (to revision d7e8b7939b554e8fdc6815da5fa349bf7d5d8aaa) to /tmp/pip-install-6sc1x528/django-s3-folder-storage_e43dd85c4bc1431ca499f78ae9634759\n\u001b[91m  Running command git clone --filter=blob:none --quiet https://github.com/pythonbrasil/django-s3-folder-storage.git /tmp/pip-install-6sc1x528/django-s3-folder-storage_e43dd85c4bc1431ca499f78ae9634759\n\u001b[0m\u001b[91m  fatal: could not read Username for 'https://github.com': No such device or address\n\u001b[0m\u001b[91m  error: subprocess-exited-with-error\n  \n  × git clone --filter=blob:none --quiet https://github.com/pythonbrasil/django-s3-folder-storage.git /tmp/pip-install-6sc1x528/django-s3-folder-storage_e43dd85c4bc1431ca499f78ae9634759 did not run successfully.\n  │ exit code: 128\n  ╰─> See above for output.\n  \n  note: This error originates from a subprocess, and is likely not a problem with pip.\n\u001b[0m\u001b[91merror: subprocess-exited-with-error\n\n× git clone --filter=blob:none --quiet https://github.com/pythonbrasil/django-s3-folder-storage.git /tmp/pip-install-6sc1x528/django-s3-folder-storage_e43dd85c4bc1431ca499f78ae9634759 did not run successfully.\n│ exit code: 128\n╰─> See above for output.\n\nnote: This error originates from a subprocess, and is likely not a problem with pip.\n\u001b[0mRemoving intermediate container 8a633591c25b\n","stderr":"The command '/bin/sh -c pip install --upgrade pip && pip install -r /tmp/requirements_test.txt' returned a non-zero code: 1\n"},"endTime":1672771592603}